src/Files.App/UserControls/Pane/PreviewPane.xaml

```diff
@@ -269,6 +271,63 @@
 					</ItemsControl.ItemTemplate>
 				</ItemsControl>
 
+				<!--  Tags List  -->
+				<StackPanel
+					x:Name="DetailsTagsList"
+					Spacing="4"
```

Comment:
Can x:Load be false when there aren't any tags? Users will still be able to add tags from the menu flyout.